metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -955,6 +956,25 @@ class MetalsLanguageServer(
                 case _ =>
                   Future.successful(())
               }
+          } else if (
+            userConfig.ammoniteJvmProperties != old.ammoniteJvmProperties && buildTargets.allBuildTargetIds
+              .exists(Ammonite.isAmmBuildTarget)
+          ) {
+            languageClient
+              .showMessageRequest(AmmoniteJvmParametersChange.params())
+              .asScala
+              .flatMap {
+                case item if item == AmmoniteJvmParametersChange.restart =>
+                  ammonite
+                    .stop()
+                    .asScala
+                    .flatMap { _ =>
+                      ammonite.start()
+                    }
+                    .ignoreValue
```

Comment:
Not needed, `ammonite.start()` already returns `Future[Unit]`

Reply:
Removed!